libcontainer/rootfs_linux_test.go

```diff
@@ -32,14 +32,6 @@ func TestCheckMountDestFalsePositive(t *testing.T) {
 	}
 }
 
-func TestCheckMountRoot(t *testing.T) {
-	dest := "/rootfs"
-	err := checkMountDestination("/rootfs", dest)
-	if err == nil {
-		t.Fatal(err)
-	}
-}
```

Comment:
I mean to leave the test (just change `err == nil` to `err != nil`). In fact, we could add an integration test for mounting over `/` because I get the feeling there's probably quite a few places that we aren't testing with those semantics.